huaweicloud/services/lts/resource_huaweicloud_lts_register_kafka_instance.go

```diff
@@ -0,0 +1,152 @@
+package lts
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var nonUpdatableParams = []string{"instance_id", "kafka_name", "connect_info", "connect_info.*.user_name", "connect_info.*.pwd"}
```

Comment:
```suggestion
var registerKafkaNonUpdatableParams = []string{"instance_id", "kafka_name", "connect_info", "connect_info.*.user_name", "connect_info.*.pwd"}
```
Avoid this vairable name conflict with others.